app/views/devise/registrations/new.html.slim

```diff
@@ -1,23 +1,90 @@
 section.sign-up class=(@have_users ? '' : 'first-user')
-  .center-panel
+  .center-panel#signup_page
     .col-md-4.col-sm-2.col-xs-1
     .col-md-4.col-sm-8.col-xs-10.text-center
       = render 'shared/notifications'
       = image_tag 'layout/portus-logo-login-page.png', class: 'login-picture'
       = form_for(resource, as: resource_name, url: { action: 'create' }, :html => {:autocomplete => "off"}) do |f|
-        = f.text_field :username, class: 'form-control input-lg', placeholder: 'Username', required: true, autofocus: true
-        = f.email_field :email, class: 'form-control input-lg first', placeholder: 'Email', required: true
-        = f.password_field :password, class: 'form-control input-lg', placeholder: 'Password (8 characters min.)', required: true
-        = f.password_field :password_confirmation, class: 'form-control input-lg last', placeholder: 'Password again', required: true
+        = f.text_field :username, class: 'form-control input-lg', placeholder: 'Username', required: true, autofocus: true,rel: 'popover'
+        = f.email_field :email, class: 'form-control input-lg first', placeholder: 'Email', required: true, rel: 'popover'
+        = f.password_field :password, class: 'form-control input-lg', placeholder: 'Password (8 characters min.)', required: true,rel: 'popover'
+        = f.password_field :password_confirmation, class: 'form-control input-lg last', placeholder: 'Password again', required: true,rel: 'popover'
 
         - if !@admin && @first_user_admin
           = f.hidden_field :admin, value: true
 
-        button.btn.btn-primary.btn-block.btn-lg type="submit"
+        button.btn.btn-primary.btn-block.btn-lg#submit_btn type="submit"
           i.fa.fa-check
           - if @admin || !@first_user_admin
             | Create account
           - else
             | Create admin
         - if @have_users
           .text-center = link_to 'I already have an account. Login.', new_user_session_path, class: 'btn btn-link'
+javascript:
```

Comment:
Move this out of the slim file. 

I suggest you to follow the same structure done on #1233. Since I was responsible for that, I can help you if you need any help.